assests/javascript/index.js

```diff
@@ -34,16 +42,87 @@ async function addTask(task) {
       body,
       headers,
     });
-    const {error} = await response.json();
+    const { error } = await response.json();
 
     if (response.status === 400) {
       showInputError(error);
+    } else {
+      showAllTasks();
     }
   } catch {
     showNetworkError();
   }
 }
 
+async function getAllTasks() {
+  try {
+    const response = await fetch(`${baseUrl}/todos`);
+    const { data } = await response.json();
+
+    return data;
+  } catch {
+    showConnecionErrorMassage();
+  }
+}
+
+async function showAllTasks() {
+  const tasks = sortTasksByTime(await getAllTasks());
+
+  taskList.innerHTML = tasks.map((task) => {
+    const { id, text, isDone, createdAt } = task;
+
+    return `<li class="task-box tooltip" id="${id}">
+              <input type="checkbox" class="task-check-box" ${isDone}></input>
+              <p class="task-title">${text}</p>
+              <i class="fa-solid fa-pen-to-square edit-task-icon"></i>
+              <i class="fa-solid fa-xmark delete-task-icon"></i>
+
+              <p class="tooltiptext">${formatDate(createdAt)}</p>
+            </li>`;
+  }).join("");
+
+  classifyTasks(tasks);
+}
+
+function formatDate(time) {
+  const dateTime = new Date(time);
+  const year = dateTime.toLocaleDateString("en-US", { year: "numeric" });
+  const month = dateTime.toLocaleDateString("en-US", { month: "short" });
+  const day = dateTime.toLocaleDateString("en-US", { day: "numeric" });
+
+  return `${day} ${month} ${year}`;
+}
+
+async function classifyTasks(tasks) {
+  const completedTasks = getCompletedTasks(tasks);
+  const uncompletedTasks = getUncompletedTasks(tasks);
+
+  numberOfAllTasks.innerText = tasks.length;
+  numberOfCompletedTasks.innerText = completedTasks.length;
+  numberOfUncompletedTasks.innerText = uncompletedTasks.length;
+}
+
+function getCompletedTasks(tasks) {
+  return tasks.filter((task) => { task.isDone });
```

Comment:
Check my previous comment carefully and explain:
1. Why this wont work.
2. How should we change it and why the change would work.
(Explain in a voice message)

Reply:
doneeee